temporalio/workflow.py

```diff
@@ -424,6 +425,14 @@ class ParentInfo:
     workflow_id: str
 
 
+@dataclass(frozen=True)
+class UpdateInfo:
+    """Information about a workflow update."""
+
+    id: str
```

Comment:
We use `workflow_id` to differentiate from `run_id` in `Info`, should we call this `update_id`?

Reply:
In Go we did ID, but `update_id` is also fine.